inc/class-wpseo-utils.php

```diff
@@ -195,34 +219,35 @@ public static function trim_recursive( $value ) {
 	 * @return string
 	 */
 	public static function translate_score( $val, $css_value = true ) {
-		if ( $val > 10 ) {
```

Comment:
My idea was to fix it in the spots that it occurs so we have a more consistent way of representing the score. Maybe it's an idea to leave this abstraction out of the current release an put it in the next?